tests/kafka/README.md

```diff
@@ -0,0 +1,17 @@
+# Kafka Sink Integration Test
+
+### Start kafka
```

Comment:
Do you have instructions for a helm chart that you've vetted against.  Given the eventrouter is a kube tool I'd like to stay in the ecosystem when possible.